README.md

```diff
@@ -63,6 +63,11 @@ creating and pinning the hook maps using the [newtools/ebpf](https://godoc.org/g
 * Display captured packets:
 `sudo xdpcap /path/to/pinned/map - "tcp and port 80" | sudo tcpdump -r -`
 
+Normally the XDP hook is loaded in legacy mode, however, if the attach
```

Comment:
This has a lot of info, and I'm afraid it might be confusing to people just looking to capture packets (eg what's legacy mode? It makes it sound like it's deprecated!).

How about just saying "Programs using `BPF_F_XDP_HAS_FRAGS` are supported, but only the first page of the packet will be captured" at the end of the "## Instrumentation" section?

Reply:
Modified the README to explain the multiple attachment tries and added a section in the limitations section to explain that we are capturing only the first page of the packet atm